performance/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@
 #
 # Mozilla Public License Version 2.0
 
+# Project include(s).
+include( traccc-compiler-options-cpp )
+include( traccc-compiler-options-cuda )
```

Comment:
`traccc-compiler-options-cpp` is a good addition (we actually need to add it in a few other places as well), but `traccc-compiler-options-cuda` is not needed here. That is only needed in subdirectories where `.cu` files are being compiled. That is not going to be the case here.

Reply:
Hi Attila. Just removed it